modules/ml/src/main/java/org/apache/ignite/ml/genetic/parameter/GAGridConstants.java

```diff
@@ -34,4 +33,5 @@
     }
 
```

Comment:
please cleanup lines 34-36

Reply:
This is they way source was formatted after applying IntelliJ's format and ignite_codeStyle.xml on source. I have corrected.